schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,257 @@
+| disease_ontology_term_id | categorical with `str` categories. This MUST be a MONDO term or [PATO:0000461](http://purl.obolibrary.org/obo/PATO_0000461) for "normal". | Curator |
+| ethnicity_ontology_term_id | categorical with `str` categories. If `organism_ontolology_term_id` is `"NCBITaxon:9606"`, this MUST be either a HANCESTRO term or `"unknown"` if unavailable. <br><br> If `organism_ontolology_term_id` is `"NCBITaxon:10090"`, this MUST be `"na"`. | Curator |
+| is_primary_data | `bool`. This MUST be `True` if this is the canonical instance of this cellular observation and `False` if not. This is commonly `False` for meta-analyses reusing data or for secondary views of data. | Curator |
+| organism_ontology_term_id | categorical with `str` categories. This MUST be either `"NCBITaxon:9606"` or `"NCBITaxon:10090"`. | Curator |
+| sex_ontology_term_id | categorical with `str` categories. This MUST be a child of "phenotypic sex". See [PATO:0001894](http://purl.obolibrary.org/obo/PATO_0001894) for details. | Curator |
+| tissue_ontology_term_id | categorical with `str` categories. This MUST be an UBERON term. This SHOULD be appended with `" (cell culture)"` or `" (organoid)"` if appropriate. | Curator |
+| | | |
+
+When a dataset is uploaded, the cellxgene Data Portal MUST automatically add the matching human-readable name for the corresponding ontology term identifier to the `obs` dataframe. Curators SHOULD NOT annotate the following columns.
+
+| Key | Value | Annotator |
+:--|:--|:--
+| assay | categorical with `str` categories. This MUST be appended with any *clarifying text* set in the matching identifier. | Data Portal |
+| cell_type | categorical with `str` categories. | Data Portal |
+| development_stage | categorical with `str` categories. This MUST be `"unknown"` if set in the matching identifier | Data Portal |
+| disease | categorical with `str` categories | Data Portal |
+| ethnicity | categorical with `str` categories. This MUST be `"na"` or `"unknown"` if set in the matching identifier. | Data Portal |
+| organism | categorical with `str` categories | Data Portal |
+| sex | categorical with `str` categories | Data Portal |
+| tissue | categorical with `str` categories. This MUST be appended with `" (cell culture)"` or `" (organoid)"` if set in the matching identifier. | Data Portal |
+| | | |
+
+## `var` (Gene Metadata)
+
+`var` is a [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html).
+
+`var.index` MUST contain unique ENSEMBL gene identifiers for features. 
+
+Curators MUST annotate the following columns in the `var` dataframe:
+
+| Key | Value | Annotator |
+:--|:--|:--
+| feature_biotype | This MUST be `"gene"`. | Data Portal |
```

Comment:
Should this be Curator instead of Data Portal in anticipation of future biotypes other than gene?

Reply:
Jason already noted this in an earlier comment. Corrected in my editor's draft.